colossalai/inference/kv_cache/kvcache_manager.py

```diff
@@ -499,6 +505,6 @@ def _init_device_caches(
         k_cache: List[torch.Tensor] = []
         v_cache: List[torch.Tensor] = []
         for _ in range(self.num_layers):
-            k_cache.append(torch.zeros(kalloc_shape, dtype=self.dtype, device=self.device))
-            v_cache.append(torch.zeros(valloc_shape, dtype=self.dtype, device=self.device))
+            k_cache.append(torch.zeros(kalloc_shape, dtype=self.kv_cache_dtype, device=self.device))
```

Comment:
`self.kv_cache_dtype` is assigned by `config.kv_cache_dtype`, which is of str type. And it will error out as `dtype` in `torch.zeros` does not expect a string.

Reply:
config dtype is a torch type beacuse of verify_config?

Reply:
Yes. And `verify_config` shall process `kv_cache_dtype` as well